### PR TITLE
Remove support of maesh env and config files

### PR DESCRIPTION
**Pull request by kevinpollet**
## What does this PR do?

This PR removes the support of maesh env and config files.

### How to test it

* make test
* make test-integration

### CODE REVIEW

File: cmd/loaders.go [hunk 1]
```diff
@@ -16,38 +16,30 @@ import (
...
+	traefikMeshVars := env.FindPrefixedEnvVars(os.Environ(), traefikMeshPrefix, cmd.Configuration)
 
-	traefikMeshVars := findAndNormalizePrefixedEnvVars(traefikMeshPrefix, cmd.Configuration)
-	maeshVars := findAndNormalizePrefixedEnvVars(maeshPrefix, cmd.Configuration)
+	var meshVars []string
 
-	if len(maeshVars) > 0 && len(traefikMeshVars) > 0 {
-		return false, fmt.Errorf("environment variable prefixed by %q cannot be mixed with variable prefixed by %q", maeshPrefix, traefikMeshPrefix)
+	for _, v := range traefikMeshVars {
+		meshVars = append(meshVars, strings.Replace(v, traefikMeshPrefix, meshPrefix, 1))
```
Comment (jspdown) [inline cmd/loaders.go:34]: Out of curiosity, why do we need to replace `traefikMeshPrefix` by `meshPrefix`? What would be the difference with just passing `traefikMeshPrefix` to `env.Decode`?

Comment (kevinpollet) [reply cmd/loaders.go:34]: Because it is currently not possible to have a prefix with multiple "_" in paerser.

**Review by dtomcej — APPROVED** [review]
LGTM
:shipit:

**Review by jspdown — APPROVED** [review]
LGTM :rocket:

**Pull request merged**